docs/governance-strategy.md

```diff
@@ -0,0 +1,137 @@
+```
+
+The MyST Specification is changed through the MyST Enhancement Proposal process, described below.
+
+### `executablebooks/myst-eps`: MyST Enhancement proposals
+
+Because the MyST specification has many interior and exterior stakeholders, we use a more formal and structured process around changing the specification. This process is encoded in `executablebooks/myst-eps` and described briefly below
+
+1. **Open an issue in `myst-eps`** that describes the enhancement you'd like to make. The goal of the issue is to help others understand your idea and get informal alignment around it. _MEPs should ideally have at least two, and ideally 3-4 co-authors from different organizations._
+2. **Make a proposal via a PR to `myst-eps`**. Use a markdown template to structure your proposal. The `status` should initially be set to `Draft`. Here's a proposed template:
+   
+   ```
+   ---
+   mep:
+      id: <NNNN - increment last active MEP # by 1>
+      created: <yyyy-mm-dd - date MEP is active>
+      authors:
+        - <authors' real names, optional github handle>
+      status: <Draft | Active | Accepted | Not Accepted>
+      discussion: <URL of canonical location for discussion>
+   ---
+   # <title here>
+
+   ## Context
+   
+   <!-- provide context needed to understand this proposal. Describe the problem with MyST's current syntax or behavior. -->
+   
+   ## Proposal
+   
+   <!-- describe your proposed change to syntax in concrete terms. Include a layperson's description of this change if relevant. -->
+   
+   ## Examples
+   
+   <!-- provide examples of what this change would look like in the real world (e.g., raw MyST and rendered output). -->
+   
+   ## UX implications
+   
+   <!-- describe how this would improve the UX or functionality of MyST markdown. Describe any deprecations or syntax migration steps that would be needed. -->
+   
+   ## Questions or objections
+   
+   <!-- as conversation takes place, list anything that is needed to be resolved and provide links to the conversation where this happened. -->
+   
+   ## References
+   
+   <!-- reference other examples you're using for inspiration or to help others learn and understand the proposal. -->
+   ```
+3. **Iterate on proposal**. Actively invite discussion from others in the community. As individuals (particularly core team members) raise objections or make suggestions, incorporate their ideas as changes into the proposal.
+4. **Make a decision**. Once the proposal has stabilized and the author wishes to move forward, make a commit to the PR to set the status as `Active`.
+   - The MEP should no-longer change substantially and **the Core Team** should review and approve if they wish.
+   - To approve, then `Approve` the Pull Request in the GitHub UI.
+   - To request blocking changes, then `Request Changes` in the GitHub UI.[^blocking]
+   - A MEP may be accepted when:
+     - More than 2 working days have passed since the proposal was marked as `Active`.
```

Comment:
I think 2 days might be a bit short here, and I don't think we necessarily need that much speed in this aspect of our process. Perhaps 5-7 days as a cut off, which gives at least some time for reflection and discussion as opinions form and/or change.

Reply:
I thought that too when I first read it, but I haven't been involved with that process so didn't weigh in on it.

Reply:
+1 I think a week is a good idea for public consideration of an MEP.

Reply:
That sounds good to me - my first thought was to keep it short so that the Core Team felt they had the agility needed to evolve the spec rapidly if need be, but I'm definitely fine with lengthening that process. The MyST EPs process in particular is a broader team process, so I welcome suggestions from that team on iterative ways we can improve the text here.